schema/schema.json

```diff
@@ -0,0 +1,348 @@
+                    "type": "object"
+                }
+            },
+            "unevaluatedProperties": false,
+            "required": [
+                "administrators",
+                "client_uuid",
+                "access_policy",
+                "pscheduler_service",
+                "name"
+            ]
+        },
+        "service": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "service name",
+                    "type": "string"
+                },
+                "locators": {
+                    "description": "service locator uris",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "uri"
+                    }
+                },
+                "version": {
+                    "description": "service version",
+                    "type": "string"
+                },
+                "meta": {
+                    "type": "object"
+                }
+            },
+            "required": [
+                "name",
+                "locators"
+            ]
+        },
+        "pscheduler_service_schema": {
+            "type": "object",
+            "properties": {
+                "tools": {
+                    "description": "pscheduler tools installed",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "tests": {
+                    "description": "pscheduler tests enabled on the host",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/$definitions/service"
+                }
+            ],
+            "unevaluatedProperties": false,
+            "required": [
+                "tools",
+                "tests"
+            ]
+        },
+        "archive_service_schema": {
+            "allOf": [
+                {
+                    "$ref": "#/$definitions/service"
+                }
+            ],
+            "unevaluatedProperties": false
+        },
+        "person": {
+            "type": "object",
+            "properties": {
+                "emails": {
+                    "description": "perfsonar host admin emails",
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "email"
+                    }
+                },
+                "meta": {
+                    "type": "object"
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "emails"
+            ]
+        }
+    },
+    "properties": {
+        "interface_capacity": {
```

Comment:
You can remove Interface_ from all of these properties since it is an interface object, so that's implied.